microsoft-edge/devtools-guide-chromium/whats-new/whats-new-archive.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
need entry in toc.yml, need indent old whatsnews in toc:
```
            - name: Microsoft Edge 101
              href: devtools-guide-chromium/whats-new/2022/04/devtools-101.md

            - name: Microsoft Edge 100
              href: devtools-guide-chromium/whats-new/2022/03/devtools-100.md

            - name: What's new archive
              items:
                - name: What's new archive
                  href: devtools-guide-chromium/whats-new/whats-new-archive.md
                  displayName: What's new in Microsoft Edge DevTools Archives # full title

                - name: Microsoft Edge 99
                  href: devtools-guide-chromium/whats-new/2022/03/devtools.md

                - name: Microsoft Edge 98
                  href: devtools-guide-chromium/whats-new/2022/02/devtools.md

                - name: Microsoft Edge 97
                  href: devtools-guide-chromium/whats-new/2022/01/devtools.md

                - name: Microsoft Edge 96
                  href: devtools-guide-chromium/whats-new/2021/11/devtools.md

                - name: Microsoft Edge 95
                  href: devtools-guide-chromium/whats-new/2021/10/devtools.md

                - name: Microsoft Edge 94
                  href: devtools-guide-chromium/whats-new/2021/09/devtools.md

                - name: Microsoft Edge 93
                  href: devtools-guide-chromium/whats-new/2021/07/devtools.md

                - name: Microsoft Edge 92
                  href: devtools-guide-chromium/whats-new/2021/05/devtools.md

                - name: Microsoft Edge 91
                  href: devtools-guide-chromium/whats-new/2021/04/devtools.md

                - name: Microsoft Edge 90
                  href: devtools-guide-chromium/whats-new/2021/02/devtools.md

                - name: Microsoft Edge 89
                  href: devtools-guide-chromium/whats-new/2021/01/devtools.md

                - name: Microsoft Edge 88
                  href: devtools-guide-chromium/whats-new/2020/11/devtools.md

                - name: Microsoft Edge 87
                  href: devtools-guide-chromium/whats-new/2020/10/devtools.md

                - name: Microsoft Edge 86
                  href: devtools-guide-chromium/whats-new/2020/08/devtools.md

                - name: Microsoft Edge 85
                  href: devtools-guide-chromium/whats-new/2020/06/devtools.md

                - name: Microsoft Edge 84
                  href: devtools-guide-chromium/whats-new/2020/05/devtools.md

                - name: Microsoft Edge 83
                  href: devtools-guide-chromium/whats-new/2020/03/devtools.md

                - name: Microsoft Edge 81
                  href: devtools-guide-chromium/whats-new/2020/01/devtools.md

                - name: Microsoft Edge 80
                  href: devtools-guide-chromium/whats-new/2019/12/devtools.md
```

Reply:
added to the toc.yml

Reply:
I accepted your suggestions